Update runtime-api.md (#312)
JoshOrndorff/recipes: text/3-entrees/runtime-api.md

about this in the docs for
[`impl_runtime_apis!`](https://substrate.dev/rustdocs/v2.0.0-rc3/sp_api/macro.impl_runtime_apis.html).

The type parameters can of course be extended with more, custom types with the only rule being that

> **the type should implement/ be bound by `parity_scale_codec::Decode`**. The following example shows how to make use

Reviewer: ```suggestion
**the type should implement/ be bound by `parity_scale_codec::Codec`**. The following example shows how to make use
```

of an extra type parameter in your runtime API.

```rust
sp_api::decl_runtime_apis! {

> 	pub trait SumStorageApi<MyTypeA> where MyTypeA: Codec {

Reviewer: ```suggestion
	pub trait SumStorageApi<MyTypeA: Codec> {
```

		fn get_sum() -> u32;
		fn do_other(cargo: MyTypeA) -> MyTypeA;
	}
}
```

> Note that **`Block`** is still added to the final implementation of the runtime.

Reviewer: ```suggestion
Note that **`Block`** is still added as first parameter to the trait declaration, thus you will need to specify it when using the trait:
```
